frontend/src/app/shared/restaurants.service.ts

```diff
@@ -31,7 +31,9 @@ export class RestaurantsService {
         lat: restaurant.location.lat,
         lng: restaurant.location.lng,
         photos: photos,
-        photo: photos[0]
+        photo: photos[0],
+        routeDistance: restaurant.route.distance + "m",
+        routeDuration: (restaurant.route.duration / 60) + "min"
       }
```

Comment:
tests are failing because we have a test that map the json received from the backend to typescript Restaurant object.
you need to add those fields in test

Reply:
let me know if you need help with anything 😺